framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/WorkbenchDeploymentImpl.java

```diff
@@ -70,19 +73,25 @@ public void setPassword(String password) {
 
     @Override
     public String getServiceName() {
-        return serviceName;
-    }
-
-    public void setServiceName(String applicationName) {
-        this.serviceName = applicationName + "-buscentr";
+        // Try to guess Workbench service name from all available services
```

Comment:
Since it's used at 2 places, I'd rather define the regexes and these helper methods in separate util class..

Reply:
Adjusted